vector/src/main/res/layout/fragment_ftue_reset_password_input.xml

```diff
@@ -115,14 +115,12 @@
             android:layout_height="wrap_content"
             app:constraint_referenced_ids="entrySignOutAll,signOutAllLabel" />
 
-        <!-- Tchap: checked by default -->
         <CheckBox
             android:id="@+id/entrySignOutAll"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginStart="-14dp"
             android:buttonTint="@color/checkbox_tint_selector"
-            android:checked="true"
```

Comment:
je ne retirerai pas le commentaire, cela aide pendant les merge

Reply:
j'ai restauré le code pour qu'il soit identique à la code base. Il faut quand même laisser le commentaire dans ce cas ?

Reply:
Si le code est identique a celui d'Element pas besoin de commentaire

En revanche, il me semble que le souhait etait de personaliser le texte associé a cette check box, non? Si on peut garder le texte actuel cela me convient. Je pose la question pour aligner le changement sur iOS

Reply:
related to https://github.com/tchapgouv/tchap-android/issues/803

Reply:
Il serait bon d'uniformiser les textes basé sur ce qui a été validé, ca simplifie le support et la documentation